texar/modules/encoders/gpt2_encoder.py

```diff
@@ -62,6 +64,8 @@ def __init__(self,
                  cache_dir: Optional[str] = None,
                  hparams=None):
 
+        self.model_name = "GPT2"
```

Comment:
Let's make it a class attribute so we can access `model_name` without having to instantiate the model:
```python
class GPT2Encoder(...):
	model_name = "GPT2"

	def __init__(...): ...
```